README.md

```diff
@@ -74,6 +74,26 @@ To create a production build, with minified and unminified output:
 npm run build:prod
 ```
 
+## Loading tota11y as an unpacked Chrome extension
+
+Build the project
+
+```
+npm run build
```

Comment:
build or build:prod?

Reply:
Ahh, I see you made some changes about a month ago. I'd go with `npm run `build:prod`. I'll update the documentation.